src/components/Video/Video.module.scss

```diff
@@ -1,6 +1,25 @@
 .frame {
   position: relative;
   overflow: hidden;
+  padding: 0;
```

Comment:
Can be removed as 0 is the default and the value doesn't change

Reply:
Fixed!